File: archive/2019/solutions/CandyBowl.cpp
#include <iostream>
using namespace std;

int main() {
	int n;
	cin >> n;
	for(int i = 0; i<n; i++){
		int x;
		cin >> x; 
		x%= 6;
		if(2<= x && x<= 4) cout << "YES" << endl;
		else cout << "NO" << endl;
	}
	return 0;
}

File: archive/2019/solutions/CandyBowl.java
import java.util.Scanner;

public class CandyBowl {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		for (int i=0; i<n; i++) {
			int a = sc.nextInt();
			if ((a % 2 == 0) != (a % 3 == 0))
				System.out.println("YES");
			else
				System.out.println("NO");
		}
		
		sc.close();

	}

}